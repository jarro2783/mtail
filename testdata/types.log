37
12.8
