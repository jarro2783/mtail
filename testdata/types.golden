gauge should_be_int 37 2017-06-15T18:03:14Z
gauge should_be_float 12.8 2017-06-15T18:03:14Z
